.gitignore

```diff
@@ -36,3 +36,9 @@ operator/tests/e2e-v2-helm/
 
 # Go releaser artifacts
 dist/
+
+# Ignore depedencies
+charts/*/charts/*.tgz
+
+# Dep Chart Lock file
+Chart.lock
```

Comment:
I'm still pretty certain we don't want to ignore Chart.lock files. That will mess with the release process and cause our subcharts to be at an unstable version.

Reply:
Ok I will remove it